app/controllers/api/quotes_controller.rb

```diff
@@ -1,15 +1,14 @@
 class Api::QuotesController < ApplicationController
-  API_URL = 'https://api.tronalddump.io'.freeze
+  API_URL = 'https://api.tronalddump.io'
 
   def show
-    if params[:id] == 'random'
+    if params[:id] == "random"
```

Comment:
In ruby, use single quotes unless you string interpolate

Reply:
We changed them to single quotes.